addons/mail/static/src/components/messaging_menu/messaging_menu.scss

```diff
@@ -31,6 +31,12 @@
         min-height: 50px;
         max-height: 400px;
         z-index: 1100; // on top of chat windows
+        @media (min-height: 600px) {
+            max-height: 520px;
+        }
+        @media (min-height: 800px) {
+            max-height: 630px;
+        }
```

Comment:
This could be made more dynamic than arbitrary breakpoints with a combination of `calc` and `min`
Something like this (values to tweak according to what we want):

`max-height: min(calc(100vh - 140px), 630px);`